modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java

```diff
@@ -254,11 +359,18 @@ public JSImportMapsRegistration addingService(
 						ServiceReference<JSImportMapsContributor>
 							serviceReference) {
 
+						Long companyId = (Long)serviceReference.getProperty(
+							"com.liferay.importmaps.company");
```

Comment:
most likely we'll like to name this `com.liferay.importmaps.companyId`

Reply:
Nope -> https://github.com/liferay/liferay-portal/blob/ed4b8edd9107f22f20f15f083e94a682a7309bae/modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/CustomElementCETPortlet.java#L55

It's `com.liferay.portlet.company` for portlets so `com.liferay.importmaps.company` for import maps :shrug: